src/selectors/per-thread/markers.js

```diff
@@ -74,11 +74,14 @@ export function getMarkerSelectorsPerThread(threadSelectors: *) {
 
   const _getFirstSampleTime: Selector<Milliseconds> = state =>
     threadSelectors.getThread(state).samples.time[0] || 0;
+  const _getLastSampleTime: Selector<Milliseconds> = state =>
+    threadSelectors.getThread(state).samples.time.slice(-1)[0] || 0;
```

Comment:
I find this way of getting the last element either clever or too clever. But let's say it's good like this :)

Reply:
Yeah, it's close to being too clever. But I'm thinking it's not too bad since those two methods are next to each other and helps make it clear what it's doing.